charts/rancher-monitoring/v0.0.4/questions.yml

```diff
@@ -1 +1,2 @@
 rancher_min_version: 2.3.0-rc1
+rancher_max_version: 2.3.4-rc1
```

Comment:
Why are the monitoring max versions not set as 2.3.3?

Reply:
fixed